changelog.d/12867.bugfix

```diff
@@ -0,0 +1 @@
+Fix documentation incorrectly stating the `sendToDevice` endpoint can be directed at generic workers. Contributed by Nick @ Beeper.
```

Comment:
We have a changelog entry type just for documentation changes -> https://matrix-org.github.io/synapse/latest/development/contributing_guide.html#changelog

I'll quickly update this before merging.